lib/apis/gravity.js

```diff
@@ -4,8 +4,8 @@ import config from '../../config';
 
 const { GRAVITY_API_BASE } = process.env;
 
-export default (path, accessToken) => {
+export default (path, accessToken, fetchOptions = {}) => {
```

Comment:
This was b/c you want some options (in this case, namely the `method: 'POST'` option) to be exposed to `fetch`.